Makefile

```diff
@@ -142,6 +142,25 @@ frontend-lint: node_modules/.uptodate
 bddtests: python
 	@tox -qe bddtests
 
+PIP_COMPILE := pyenv exec pip-compile
```

Comment:
Using a variable for the compiler is just a `Makefile` convention. It means that if you ever need to you can install some other version of `pip-compile` somewhere else and do `make requirements PIP_COMPILE=<path_to_other_pip_compile>`